app/models/workflow_execution.rb

```diff
@@ -11,6 +11,7 @@ class WorkflowExecution < ApplicationRecord
 
   has_many :samples_workflow_executions, dependent: :destroy
   has_many :samples, through: :samples_workflow_executions
+  has_many :outputs, dependent: :nullify, class_name: 'Attachment', as: :attachable
```

Comment:
**nitpick (blocking):** I think we can remove the `dependent: :nullify` since if the `WorkflowExecution` is destroyed it should destroy the `Attachment` instances.

Reply:
fixed in https://github.com/phac-nml/irida-next/pull/439/commits/4aecb110d87bcdc9df22480b43862175567669ad